src/@hooks/useAddressConfig.ts

```diff
@@ -67,13 +68,22 @@ export function useAddressConfig(): UseAddressConfig {
     )
   }
 
+  const getVerifiedAddressName = (address: string) => {
+    const addressKey = Object.keys(verifiedAddresses).find((key) => {
+      return key.toLowerCase() === address.toLowerCase()
+    })
+    const addressName = verifiedAddresses[addressKey]
+    return addressName?.length > 0 ? addressName : undefined
```

Comment:
suggestion: this return statement now be shortened
```suggestion
    return addressName
```